src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
```

Comment:
Constructor parameters 'line' and 'comment' are directly assigned to class fields, lacking input validation. (Severity: Low)